### DISCUSSION TITLE
"input and output interfaces missing" when use pmacct on linux exporter

### DISCUSSION THREAD

**Discussion by manuelpiacenti** [akvorado/akvorado#1499] [Q&A]
I am currently running a Debian 11 virtual machine, where I have installed the pmacct package to forward NetFlow streams to my Akvorado server. However, I’m encountering an issue: Akvorado displays an error stating, “input and output interfaces missing,” and as a result, the graphs are not populated.

When I run the command "pmacctd -a" on vm Debian11, it appears that the in_iface and out_iface aggregates are not available, which may be causing this problem.

Would there be a workaround on Akvorado to bypass this error, or would you recommend a different setup for exporting NetFlow data from my Debian VM? I’m open to suggestions if there’s a more compatible tool or configuration that could resolve this issue.

Thank you for your time and assistance.

```
akvorado_inlet_core_flows_errors_total{error="SNMP cache miss",exporter="192.168.156.162"} 222833
akvorado_inlet_core_flows_errors_total{error="input and output interfaces missing",exporter="192.168.156.162"} 496335
```

**Comment by vincentbernat** [comment]
Check the documentation to use pmacctd: https://demo.akvorado.net/docs/operations#gnulinux

**Reply by manuelpiacenti** [reply]
Thank you for your response.

I’ve adjusted the configuration, and I am no longer seeing the “input and output interfaces missing” error. However, I am now encountering the “SNMP cache miss” error. Additionally, from the web interface, I don’t see any data populated under “last flow,” nor are the graphs being generated.

Below is the output of the command:
```
curl -s http://192.168.156.112:8081/api/v0/inlet/metrics | grep 192.168.156.162
``` 
Thank you in advance for any guidance you can provide

<img width="1020" alt="Screenshot 2024-11-13 alle 09 57 34" src="https://github.com/user-attachments/assets/b5f69ae6-3849-4e9e-a8bd-1394846f7d51">
<img width="1266" alt="Screenshot 2024-11-13 alle 09 52 56" src="https://github.com/user-attachments/assets/37464a91-8127-4ba1-8e8f-49c22125c178">

**Reply by vincentbernat** [reply]
The cache miss may be because the traffic you locally capture is locally generated, so you are missing the input interface or output interface. This shouldn't be a problem. Continue the troubleshooting guide: https://demo.akvorado.net/docs/troubleshooting#troubleshooting:~:text=Check%20that%20flow%20are%20correctly%20accepted%20with%3A
